contrib/tools/relay-server/README.md

```diff
@@ -0,0 +1,13 @@
+# A Relay Server
+
+## Start the `relay-server` server
```

Comment:
Can you add a description of this application:
- What does it do?
- What port does it listen to?
- How does it relay messages?